src/libraries/System.Linq/README.md

```diff
@@ -0,0 +1,10 @@
+# System.Linq
+Documentation can be found at https://learn.microsoft.com/dotnet/api/system.linq.
+
+## Contribution Bar
+- [x] [We consider new features, new APIs and performance changes](../../libraries/README.md#primary-bar)
```

Comment:
I would probably also add that we're accepting analyzers, see https://github.com/dotnet/runtime/tree/main/src/libraries/System.Collections#contribution-bar

Reply:
```suggestion
- [x] [We consider new features, new APIs and performance changes](../../libraries/README.md#primary-bar)
- [x] [We consider PRs that target this library for new source code analyzers](../../libraries/README.md#secondary-bars)
- [x] [We don't accept refactoring changes due to new language features](../../libraries/README.md#secondary-bars)
```

Should we match this then? Apply suggestion if looks good

Reply:
I'd probably remove the entry on new language features, since this library is closely tied to the language itself.